packages/gatsby/src/redux/index.ts

```diff
@@ -28,6 +27,8 @@ export const readState = (): IGatsbyState => {
         if (!state.nodesByType.has(type)) {
           state.nodesByType.set(type, new Map())
         }
+        // The `.has` and `.set` calls above make this safe
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
is this necassary since we have that bang still? do we need to remove the bang? I realize it's safe, that's why I put the bang in there.

Reply:
`@typescript-eslint/no-non-null-assertion` has warning level, so it doesn't impact overall result of `lint`, just removes particular warning for this line

Reply:
I added the comment to suppress the warning and make it clear why this is ok